contrib/http_util/src/main/java/io/opencensus/contrib/http/util/HttpTraceUtil.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opencensus.contrib.http.util;
+
+import io.opencensus.trace.Status;
+import javax.annotation.Nullable;
+
+/**
+ * A helper class to provide convenience methods for tracing.
+ *
+ * @since 0.13.0
+ */
+public final class HttpTraceUtil {
+
+  private HttpTraceUtil() {}
+
+  /**
+   * Parse OpenCensus Status from HTTP response status code.
+   *
+   * <p>This method serves a default routine to map HTTP status code to Open Census Status. The
+   * mapping is defined in <a
+   * href="https://github.com/googleapis/googleapis/blob/master/google/rpc/code.proto">Google API
+   * canonical error code</a>, and the behavior is defined in <a
+   * href="https://github.com/census-instrumentation/opencensus-specs/blob/master/trace/HTTP.md">OpenCensus
+   * Specs</a>.
+   *
+   * @param statusCode the HTTP response status code. {@code 0} means invalid response.
+   * @param error the error occured during response transmission (optional).
+   * @return the corresponding OpenCensus {@code Status}.
+   * @since 0.13
+   */
+  public static final Status parseResponseStatus(int statusCode, @Nullable Throwable error) {
```

Comment:
I'd prefer to have a simpel `@Nullable String message` as second parameter. In Scala land  (or FP in general) we do not use Exceptions to signal "expected errors" (like a 404), instead we encode it into an ADT (like an Either monad). 

Having the need of an Throwable here would force everyone to use exceptions for expected errors.

Reply:
Ack. I will leave it as it is now (i.e., null indicating no error).
For `Either` or `Option` or whatever ADT, this would require an extra transform.